### PR TITLE
Clarifying what rule is being enforced.

### PR DESCRIPTION
**Pull request by kzsh**
Capitalization of constructors is not the rule in question here.  It makes it more clear what the offending difference is between `//bad` and `//good` to have consistent constructor capitalization.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1148,7 +1148,7 @@
     var OBJEcttsssss = {};
     var this_is_my_object = {};
     function c() {}
-    var u = new user({
+    var u = new User({
```
Comment (kzsh) [inline README.md]: u is bad, not the fact that "user" should be "User"

The constructor rule is shown in the next example below (PascalCase)

**Comment by goatslacker** [comment]
We should just kill this example from this section

**Comment by goatslacker** [comment]
@kzsh do you want to amend changes or shall I?

**Comment by kzsh** [comment]
Hey @goatslacker, I'll take care of it now.  Just one thought, though.  It might be worth keeping the `var u` part to be more clear that it's not just somehow applied to functions.  What about `var o = {};` after the other two?  I'll amend it that way, and if you prefer the other way, I'll amend it again.

File: README.md [hunk 2]
```diff
@@ -1147,17 +1147,12 @@
     // bad
     var OBJEcttsssss = {};
     var this_is_my_object = {};
+    var o = {};
```
Comment (goatslacker) [inline README.md]: single letter names is mentioned above

**Comment by goatslacker** [comment]
Can you rebase down to a single commit too?

**Comment by kzsh** [comment]
Ok, let me know if that works out.

**Comment by goatslacker** [comment]
Thanks :beers:

**Pull request merged**